misc/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${fileDirname}",
+            "env": {
+                "WATCH_NAMESPACE": "myproject",
+                "APP_USER": "APPLICATION_USER",
+                "APP_PASS": "APPLICATION_PASSWORD",
```

Comment:
Wouldn't it be better to update non-standard image to understand operator set-up? We want to bring them closer to upstream anyway.

Reply:
Yeah it would be better, but atm we need that defines. When images will be closer we can just remove the env vars and the operator will run with defaults